apps/server/src/modules/h5p-editor/temporary-file-storage/temporary-file-storage.spec.ts

```diff
@@ -127,58 +130,80 @@ describe('TemporaryFileStorage', () => {
 	describe('deleteFile is called', () => {
 		describe('WHEN file exists', () => {
 			it('should delete file', async () => {
-				const { user, filename, filepath } = setup();
-				await service.deleteFile(filename, user.id);
-				const fileExists = existsSync(filepath);
-				expect(fileExists).toBe(false);
+				const { user1, file1 } = setup();
+				repo.findByPath.mockResolvedValue(file1);
+				await storage.deleteFile(file1.filename, user1.id);
+				expect(repo.delete).toHaveBeenCalled();
+				expect(s3clientAdapter.delete).toHaveBeenCalledWith([join(user1.id, file1.filename)]);
 			});
```

Comment:
```suggestion
				const { user1, file1 } = setup();
				repo.findByPath.mockResolvedValueOnce(file1);
				
				await storage.deleteFile(file1.filename, user1.id);
				
				expect(repo.delete).toHaveBeenCalled();
				expect(s3clientAdapter.delete).toHaveBeenCalledWith([join(user1.id, file1.filename)]);
			});
```
We want a blank line between Arrange, Act and Assert. See "Test Structure" in our docs: https://hpi-schul-cloud.github.io/schulcloud-server/additional-documentation/nestjs-application/testing.html

And it is preferable to always use "mockResolvedValueOnce" instead of "mockResolvedValue" if possible.

Reply:
Changed tests to adhere to styleguides